lms/static/sass/features/_course-experience.scss

```diff
@@ -1,3 +1,66 @@
+// Course call to action message
+.course-message {
+  .message-author {
+    display: inline-block;
+    width: $baseline*7/2;
+    border-radius: $baseline*7/4;
+    border: 1px solid $lms-border-color;
+
+    @media (max-width: 880px) {
```

Comment:
It's an interesting point that has mixed feelings around the community. The consensus from most is that the max-width/min-width depends on 1) The css defaults (in this specific case, we would end up adding extra css because we aren't using default values) and 2) Desktop > Mobile or Mobile > Desktop, as you mentioned. I almost feel like our current development direction is from Desktop > Mobile, since the designs we get from UX are in Desktop mode and only after initial implementation do we consider the mobile version of the experience.

It seems like a sort of toss up, in this case though I'll switch it to your recommendation, as we should start thinking Mobile > Desktop, especially as we make the move to bootstrap.